### DISCUSSION TITLE
Download location?

### DISCUSSION THREAD

**Discussion by Bwangster12** [maxrave-dev/SimpMusic#373] [General]
Where is SimpMusic downloading the songs?  I was hoping to download some to upload to my Pixel Watch, but can't find where the files are located.

**Comment by luanmdlima** [comment]
I'm also looking for it.

**Reply by maxrave-dev** [reply]
The app cache, if you want to download mp3, open the info screen in Now playing screen, scroll down and click to download button

**Reply by luanmdlima** [reply]
Is there a way to get the MP3 for all my songs all at once? Downloading one by one can be quite inconvenient, since I want to download over a thousand songs.
